corehq/apps/geospatial/views.py

```diff
@@ -367,32 +366,28 @@ def get_paginated_cases_without_gps(self, domain, page, limit):
 
 def _get_paginated_users_without_gps(domain, page, limit, query):
     location_prop_name = get_geo_user_property(domain)
-    query = (
+    res = (
```

Comment:
@zandre-eng What do you think of this change?

Reply:
Apologies for the late response on this as I was away on PTO. The changes look good, and it's nice that we're able to handle pagination now directly in the ES query.